sync/class.jetpack-sync-module-woocommerce.php

```diff
@@ -81,7 +98,7 @@ public function expand_order_item_ids( $args ) {
 
 		return array(
 			$order_items,
-			$this->get_metadata( $order_item_ids, 'order_item', $this->meta_whitelist )
+			$this->get_metadata( $order_item_ids, 'order_item', $this->order_item_meta_whitelist )
```

Comment:
@westi I'd +1 @nabsul's comment here. `phpcs` will complain if you don't add one, so it's probably best to fix this:

![screen shot 2017-05-09 at 15 10 12](https://cloud.githubusercontent.com/assets/426388/25852451/a7a2764e-34c9-11e7-93e6-2f583462858e.png)

While we don't refactor code just for the sake of it, we try to adhere to [WordPress coding standards](https://github.com/Automattic/jetpack/blob/master/docs/coding-guidelines.md#general) when adding new code or editing existing lines.

Thanks!